Mathlib/NumberTheory/NumberField/Embeddings.lean

```diff
@@ -1005,6 +1007,140 @@ lemma card_eq_card_isUnramifiedIn [NumberField k] [IsGalois k K] :
       Finset.card (Finset.univ.filter <| IsUnramifiedIn K (k := k))ᶜ * (finrank k K / 2) := by
   rw [← card_isUnramified, ← card_isUnramified_compl, Finset.card_add_card_compl]
 
+variable {K : Type*} [Field K] [NumberField K] (v : InfinitePlace K)
+
+noncomputable instance : Inhabited (InfinitePlace K) :=
+  ⟨Classical.choice (instNonemptyInfinitePlaceOfNumberField K)⟩
```

Comment:
Why do you need this (and not `Nonempty`)

Reply:
This instance is used for the `Nontrivial` instance in `InfiniteAdeleRing.lean`, which itself is needed for injectivity of the global embedding.

Reply:
Then you should use `Pi.nontrivial_at` there instead. Inhabited is usually only used if there is a distinguished element.

Reply:
Gotcha, changed that and removed `Inhabited`.